src/bitwarden_exporter/utils.py

```
@@ -0,0 +1,85 @@
+                raise BitwardenException(error_msg)
+            jmespath_password = jmespath_password[0]
+
+        if not isinstance(jmespath_password, str):
+            LOGGER.info("Vault password is not a string")
+            raise BitwardenException(error_msg)
+
+        LOGGER.info("Vault password is set from JMESPath expression")
+        secret_path = jmespath_password
+
+    if os.path.exists(secret_path) and os.path.isfile(secret_path):
+        with open(secret_path, "r", encoding="utf-8") as f:
```

Ruff (0.14.2)

Unnecessary mode argument

Remove mode argument

(UP015)